lua/lspsaga/symbol/outline.lua

```diff
@@ -133,7 +133,7 @@ function ot:parse(symbols, bufnr, list)
 
       if config.outline.detail then
         buf_set_extmark(bufnr, ns, row - 1, 0, {
-          virt_text = { { node.detail or '', 'Comment' } },
+          virt_text = { { node.detail or '', 'SagaVirtLine' } },
```

Comment:
there is better use `SagaDetail` create it in highlight.lua and link to `Comment`